docs/modules/record-encryption/aws-kms/con-aws-kms-setup.adoc

```diff
@@ -38,16 +39,17 @@ aws iam create-login-profile --user-name ${ADMIN} --password ${INITIAL_PASSWORD}
 echo Now log in at ${CONSOLE_URL}  with user name ${ADMIN} password ${INITIAL_PASSWORD} and change the password.
 ----
 
-== Filter Actor
+== Filter IAM Identity
```

Comment:
I don't know what `Filter IAM Identity` means. 

I think just using `IAM Identity` as the heading is clear enough.

Reply:
"IAM Identity" alone wouldn't get across that this the identity that will be used by the Filter. How about 'IAM Identity for the Filter'?

Reply:
I used the title "Establish an application identity for the Filter"